src/dmd/dtemplate.d

```diff
@@ -8266,3 +8281,29 @@ void printTemplateStats()
         printf("%8u %8u   %s\n", ts.numInstantiations, ts.uniqueInstantiations, (cast(const TemplateDeclaration) td).toChars());
     }
 }
+
+struct Pool(T)
```

Comment:
Should go into `root` or some other utility file, not in `dtemplate.d`

Reply:
OK